sdk/metricsadvisor/ai-metrics-advisor/package.json

```diff
@@ -78,6 +78,7 @@
   "prettier": "@azure/eslint-plugin-azure-sdk/prettier.json",
   "dependencies": {
     "@azure/core-auth": "^1.1.3",
+    "@azure/identity": "^1.1.0",
```

Comment:
@KarishmaGhiya Please import `TokenCredential` from `core-auth` and not from `identity` and then remove this dependency in another PR.

Reply:
#13677 :)